jsk_panda_robot/panda_eus/euslisp/end-coords_publisher.l

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env roseus
+
+(require "package://panda_eus/euslisp/dual_panda-interface.l")
+
+(ros::roseus-add-msgs "std_msgs")
+(ros::roseus-add-msgs "sensor_msgs")
+(ros::roseus-add-msgs "geometry_msgs")
+
+(ros::roseus "pose-stamped-publisher")
+
+(setq *robot* (dual_panda))
```

Comment:
You should be able to use something like this;  (ref. #4 )
```
(load "package://panda_eus/euslisp/dual_panda-interface.l")

(defmethod dual_panda-robot-interface
  (:dummy-controller (&rest args)))
(setq *ri* (instance dual_panda-robot-interface :init
                     :type :dummy-controller))

(do-until-key
 (print (send *ri* :state :potentio-vector))

```

Also since this is useful script, could you modify such that
1. include this node in jsk_panda_startup
2. add "/dual_panda/end_coords/*arm" to `franka_record.launch`
3. Set publishing hz using rosparam (see below comments)